specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json

```diff
@@ -968,27 +1352,133 @@
             "$ref": "#/parameters/SubscriptionIdParameter"
           },
           {
-            "name": "location",
-            "in": "path",
-            "description": "the region",
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AppNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CustomDomainNameParameter"
+          },
+          {
+            "name": "domainResource",
+            "in": "body",
+            "description": "Parameters for the create or update operation",
             "required": true,
-            "type": "string"
+            "schema": {
+              "$ref": "#/definitions/CustomDomainResource"
```

Comment:
>CustomDomainResource [](start = 37, length = 20)

The PATCH request body is the same as the response, which implies all properties are patchable. Is that really true?

Reply:
that may be mistake here, let me check.

Reply:
all properties should be present for one patch, I think this may has gap for all properties are patchable. Any suggestion to indicate this concept ?

Reply:
Typically swagger authors create a separate definition for PATCH request body that only includes the properties that are patchable (usually just tags but not always). Successful PATCH response body should remain the full resource body as it is right now.

---
In reply to: [402169775](https://github.com/Azure/azure-rest-api-specs/pull/8792#discussion_r402169775) [](ancestors = 402169775)

Reply:
If that I think this is OK, all properites are patchable and actually return the whole resource as response.

Reply:
Sounds good.

---
In reply to: [402723113](https://github.com/Azure/azure-rest-api-specs/pull/8792#discussion_r402723113) [](ancestors = 402723113)